### PR TITLE
HIVE-27205: Updated jackson version for CVE 2022-42003

### PR DESCRIPTION
**Pull request by Diksha628**

### CODE REVIEW

**Comment by Diksha628** [comment]
Updated jackson version to fix CVE 2022-42003

**Comment by Diksha628** [comment]
@zabetak , @abstractdog , @ayushtkn can one of you please approve the workflow builds for this PR ?

**Comment by TuroczyX** [comment]
> @zabetak , @abstractdog , @ayushtkn can one of you please approve the workflow builds for this PR ?

It is failing :( 

 Non-resolvable import POM: Could not find artifact com.fasterxml.jackson:jackson-bom:pom:2.12.7.1 in central (https://repo.maven.apache.org/maven2) @ line 733, column 19 -> [Help 2]

**Comment by abstractdog** [comment]
here are the jackson-bom versions for instance: https://mvnrepository.com/artifact/com.fasterxml.jackson/jackson-bom
I cannot see 2.12.7.1, is it a valid version?

File: pom.xml [hunk 1]
```diff
@@ -148,7 +148,7 @@
     <httpcomponents.client.version>4.5.13</httpcomponents.client.version>
     <httpcomponents.core.version>4.4.13</httpcomponents.core.version>
     <ivy.version>2.5.1</ivy.version>
-    <jackson.version>2.12.7</jackson.version>
+    <jackson.version>2.12.7.1</jackson.version>
```
Comment (ayushtkn) [inline pom.xml:151]: You are confused with jackson-databind version and jackson bom version, 2.12.7.1 is the version for jackson-databind 
https://mvnrepository.com/artifact/com.fasterxml.jackson.core/jackson-databind/2.12.7.1

**Review by ayushtkn — CHANGES_REQUESTED** [review]
Move the version to **2.12.7.20221012**, that should solve your purpose of upgrading jackson-databind to **2.12.7.1** 

```
jackson-databind 2.12.7.1 (12-Oct-2022) -- with jackson-bom version 2.12.7.20221012
```
can read it here https://github.com/FasterXML/jackson/wiki/Jackson-Release-2.12

File: pom.xml (hunk 1 above)
Comment (Diksha628) [reply pom.xml:151]: Thanks @ayushtkn  for pointing out, will raise another PR with the right version.

**Review by ayushtkn — APPROVED** [review]
LGTM

**Comment by Diksha628** [comment]
@ayushtkn , @abstractdog , @zabetak , can one of you please merge this PR ?

**Comment by ayushtkn** [comment]
Was waiting for @abstractdog , if he doesn't have any further comments, I can merge it

**Comment by Diksha628** [comment]
Sure, @ayushtkn , is possible, can we merge it by today, if there are no further comments ?

**Comment by ayushtkn** [comment]
I think someone pushed a change to move to higher version.
@TuroczyX got that chased as well :-)

https://github.com/apache/hive/pull/4102

**Pull request closed without merging**